### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by HTQ05** [Open-Systems-Pharmacology/PK-Sim#1433]
This error is appearing when tried to change the value of Gastric emptying Time in Population Simulation
What should I do?

![Error](https://user-images.githubusercontent.com/60983521/74606339-8233e180-50f1-11ea-96fd-d4d83953e7f1.jpeg)

_Originally posted by @HTQ05 in https://github.com/Open-Systems-Pharmacology/Forum/issues/461#issuecomment-586711898_

**Comment by msevestre** [comment]
@HTQ05 so I reproduce the issue here? Create a populaton simulation, go into the hiearchy mode and update the Gastric emptying time?

**Comment by msevestre** [comment]
You are probably changing a parameter that is coming from the population. There is a reason why it is hidden in the first place. While it should not crash, changing the value will have absolutely NO effect as it will be overriden by the population valuyes

@Yuri05 DO you concur?

**Comment by Yuri05** [comment]
Agree, those parameters should be hidden in the pop hierarchy view.

**Comment by HTQ05** [comment]
Alright. Thanks
